Cargo.toml

```diff
@@ -20,8 +20,7 @@ futures-channel = "0.3"
 futures-util = { version = "0.3", default-features = false }
 http = "0.2"
 
-# Necessary to overcome msrv check of rust 1.49, as 1.15.0 failed
-once_cell = "=1.14"
+once_cell = "1.17"
```

Comment:
Do we _require_ `.17`? What's the minimum version we need? That should be what is set here. Then, anyone else who is holding back upgrades isn't forced by us.